gateway/api/user/user.go

```diff
@@ -88,8 +88,8 @@ func Create(c *gin.Context) {
 		HashedPassword: newUser.HashedPassword,
 		Picture:        newUser.Picture,
 		Email:          newUser.Email,
-		Verified:       newUser.Verified, // DEPRECATED in flavor of role
-		Status:         string(openapi.StatusActive),
+		Verified:       newUser.Verified,
```

Comment:
Is it no longer "Deprecated in favor of role"?

Reply:
no, we will keep "verified" because it's an email-verified feature from IDPs. Roles play another function on all of this